code/datums/brain_damage/schizo.dm

```diff
@@ -0,0 +1,219 @@
+#define OWNER 0
+#define STRANGER 1
+
+/datum/brain_trauma/severe/split_personality
+	name = "Split Personality"
+	desc = "Patient's brain is split into two personalities, which randomly switch control of the body."
+	scan_desc = "complete lobe separation"
```

Comment:
That wouldn't create a split personality at all, but okay.

Reply:
Makes sense to me.

Reply:
Except it makes absolutely no sense? Personalities can't be created by separating the left and right side of the brain. It could cause issues with coordination, or not being able to describe things that are seen with the opposite side of the brain relative to the language center, but not create a split personality.